docs/pages/ktextbox.vue

```diff
@@ -2,15 +2,55 @@
 
   <DocsPageTemplate apiDocs>
     <DocsPageSection title="Overview" anchor="#overview">
-      For design guidance, see the page on <DocsInternalLink href="/textfields" text="text fields" />.
+      For detailed design guidance, refer to the page on <DocsInternalLink href="/textfields" text="text fields" />.
+    </DocsPageSection>
+
+    <DocsPageSection title="Input Types and Features">
+      <h3>Input with Label</h3>
+      <KTextbox label="Input with Label" />
+      <p>This text box includes a visible label, providing clear guidance and context to the user about the expected input.</p>
+
+      <h3>Input without Label</h3>
+      <KTextbox />
+      <p>This text box does not include a visible label. It's useful when the context of the input is obvious without a label, or if space constraints necessitate omitting it.</p>
```

Comment:
I think that since the `label` prop is `required` that we should remove this one since it results in a console error.